docs/02-Install.md

```diff
@@ -167,29 +168,29 @@ More detailed specifications can be consulted in the
 - `/data/*` subdirectories were created by the imaging install script. If not,
     it may be due to `root:root` ownership of the `/data/` mount on your
     system. Ensure these subdirectories are created manually, particularly:
-    `/data/$PROJ/data/*`, `/data/$PROJ/bin/mri/` and `/data/incoming/`
+    `/data/$PROJECT/data/*`, `/data/$PROJECT/bin/mri/` and `/data/incoming/`
```

Comment:
Do you think we should explicitly name the `tarchive`, `pic`, `logs`, `trashbin`, `batch_output`, `jiv`, `assembly`  subdirectories instead of just `/data/$PROJECT/data/*`?

Reply:
done!